engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/displaytext/InexhaustivePatternMatchToDisplayTextNode.java

```diff
@@ -9,10 +9,10 @@
 import org.enso.interpreter.runtime.callable.atom.AtomConstructor;
 import org.enso.interpreter.runtime.data.text.Text;
 
-@BuiltinMethod(type = "Inexhaustive_Pattern_Match_Error", name = "to_display_text")
-public abstract class InexhaustivePatternMatchErrorToDisplayTextNode extends Node {
-  static InexhaustivePatternMatchErrorToDisplayTextNode build() {
-    return InexhaustivePatternMatchErrorToDisplayTextNodeGen.create();
+@BuiltinMethod(type = "Inexhaustive_Pattern_Match", name = "to_display_text")
+public abstract class InexhaustivePatternMatchToDisplayTextNode extends Node {
+  static InexhaustivePatternMatchToDisplayTextNode build() {
+    return InexhaustivePatternMatchToDisplayTextNodeGen.create();
   }
 
   abstract Text execute(Object self);
```

Comment:
I wonder if we could extend our `Builtin` class to auto-generate these just from a `toText` method? It is a bit tedious having to update these each time (although I guess the refactor tool deals with these OK so it's mostly tedious to review :wink: )